Formula/i/inlyne.rb

```diff
@@ -0,0 +1,41 @@
+class Inlyne < Formula
+  desc "GPU powered yet browserless tool to help you quickly view markdown files"
+  homepage "https://github.com/Inlyne-Project/inlyne"
+  url "https://github.com/Inlyne-Project/inlyne/archive/refs/tags/v0.4.1.tar.gz"
+  sha256 "62e089627812690b182520138708151f47391c5ba518f0b80649942dedd1be87"
+  license "MIT"
+  head "https://github.com/Inlyne-Project/inlyne.git", branch: "main"
+
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on :macos # currently, the linux build has some issue, notified upstream about it
+
+  uses_from_macos "expect" => :test
```

Comment:
Can it not work with the PTY?

Reply:
yeah, it does not. I tried `w.write "q"` it did not work.